src/components/TodoItem.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import './todo.css';
+import '@fortawesome/fontawesome-free/css/all.min.css';
```

Comment:
Instead of disabling the linters, it would be more beneficial to fix them and ensure their proper functioning.